src/typing/flow_error.ml

```diff
@@ -1413,3 +1415,9 @@ let rec error_of_msg ~trace_reasons ~op ~source_file =
         null_loc, ["Potentially null/undefined value."];
         falsy_loc, [spf "%s value." value_str]
       ]]
+  | EUnknownProperty (loc, name) ->
+    mk_error
+      ~kind:(LintError LintSettings.UnknownProperty)
```

Comment:
The case that has me nervous is the refinement of a mixed type: https://github.com/facebook/flow/blob/master/tests/refinements/cond_prop.js#L23

Everything else is more or less expected.

My sense is that many developers are used to this pattern, and it should be excluded from the rule. What do you think the way forward should be?